playbooks/k8s-cluster/nfs-client-provisioner.yml

```diff
@@ -0,0 +1,41 @@
+---
+# This playbook is used to setup the nfs-client-provisioner
+# If the cluster already has an nfs server setup run with --skip-tags="nfs_server,nfs_mkdir"
+#   Otherwhise an NFS server will be configured on kube-master[0] 
+# After settting up an NFS server, nfs software is installed on all nodes
+# Lastly, the nfs-client-provisioner is helm installed using the role
+#
+# Install the required NFS software on all nodes and then helm install the nfs-client 
+- hosts: kube-master[0]
+  tasks:
+  - name: Make sure NFS directory exists and is globally writable
+    file:
+      path: "{{ k8s_nfs_export_path }}"
+      state: directory
+      mode: "u=rwx,g=rwx,o=rwx"
```

Comment:
I'm not sure if this is the best thing to do here, but it seemed necessary based on all the different container users.